client_dashboard/common/src/commonMain/kotlin/org/thechance/common/domain/getway/IRemoteGateway.kt

```diff
@@ -9,11 +9,11 @@ interface IRemoteGateway {
 
     fun getUsers(page: Int, numberOfUsers: Int): DataWrapper<User>
 
-    suspend fun getTaxis(): List<Taxi>
+    suspend fun getTaxis(page: Int, numberOfUsers: Int): DataWrapper<Taxi>
 
-    suspend fun createTaxi(taxi: AddTaxi): Taxi
+    suspend fun createTaxi(taxi: NewTaxiInfo): Taxi
 
-    suspend fun findTaxiByUsername(username: String): List<Taxi>
+    suspend fun findTaxiByUsername(username: String, page: Int, offset:Int): DataWrapper<Taxi>
```

Comment:
find taxi from its name should return one taxi why its returning datawrapper ??

Reply:
change it to more convienient name

Reply:
but i find one taxi in the end when i insert username 
username is unique